Misc/NEWS.d/next/Library/2022-09-05-17-08-56.gh-issue-72719.jUpzF3.rst

```diff
@@ -0,0 +1,3 @@
+Remove modules :mod:`asyncore` and :mod:`asynchat`, which were deprecated by :pep:`594`.
+
+They are temporarily moved to :mod:`test.support` so some other tests that rely on them keep working.
```

Comment:
I suggest to remove the second sentence. I prefer to hide them as well as possible :-)

Reply:
Done, thank you for the review!